test/strategies.test.ts

```diff
@@ -0,0 +1,63 @@
+import * as fc from '../src/index'
+import {it} from 'mocha'
+import {expect} from 'chai'
+
+describe('Strategy tests', () => {
+  describe('Constants extraction tests', () => {
+    it('should be able to extract constant that is compromising the addition commutative property', () => {
+      expect(fc.scenario()
+        .config(fc.strategy()
+          .withRandomSampling()
+          .withBias()
+          .withSampleSize(100)
+          .withConstantExtraction()
+        )
+        .forall('a', fc.integer())
+        .forall('b', fc.integer())
+        .then(({a, b}) => a + b === 100 ? a + b === b : a + b === b + a)
+        .check()
+      ).to.have.property('satisfiable', false)
+    })
+
+    it('should be able to extract constants and compute numeric range from assertion', () => {
```

Comment:
I still think that your extraction strategy being able to solve this case is too much magic and might come back to bite us in the ass later on.

Reply:
@arestivo According to your opinion and also with what I have discussed with @hugoferreira at our last meeting I have decided to make this PR as simple as possible and therefore removed the numeric range computation as well as the string mutation. 

As such, the constants extraction is simple as extracting numbers and strings, where for numbers, it extracts the value and creates two other values that are immediately above and below, and for strings, it extracts the value and creates two other strings that are the substrings of the original one and with half the size.

The developed code for string manipulation and numerics range computation (now removed) was not completely deleted because I decided to save it in my gists for future use if needed. 

The idea now (after this PR being merged) is to create a synthetic test suite related to constants extraction and as we identify new necessities we expand/enhance the constant extraction feature.